src/tests/ftest/util/general_utils.py

```diff
@@ -377,12 +377,14 @@ def get_file_path(bin_name, dir_path=""):
     Raises:
         OSError: If failed to find the bin_name file
     """
+    log = getLogger()
     with open('../../.build_vars.json') as json_file:
         build_paths = json.load(json_file)
     basepath = os.path.normpath(build_paths['PREFIX'] + "/../{0}"
                                 .format(dir_path))
 
     file_path = list(Path(basepath).glob('**/{0}'.format(bin_name)))
+    log.info("+++DM: get_file_path.file_path = {}".format(file_path))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments